### DISCUSSION TITLE
Read parameter values from list/csv-file

### DISCUSSION THREAD

**Discussion by patrick-pd** [ansys/pyaedt#1862] [Q&A] [answered]
Hi,

so my goal is to have a multipart scenario and conducting doppler analysis like in the example "SBR+: doppler setup" (Link: https://aedt.docs.pyansys.com/examples/02-HFSS/SBR_Doppler_Example.html) from the documentation. But I don't want to define the trajectory of the objects mathematically and use the time variable to update the poses. I already have a list of poses for my objects over time (as result from another simulation). What I want to do is to use the time variable in HFSS sort of as index to retreive the next values from my lists and update the parameters of my objects in HFSS.
I found something in the documentation that might be what I am looking for (Link: https://aedt.docs.pyansys.com/API/_autosummary/pyaedt.application.Variables.CSVDataset.html#pyaedt.application.Variables.CSVDataset). So let's say I have my data organized in a CSV-file and I want to update the position and orientation of the objects in the "SBR+: doppler setup" example by reading the new pose values from the CSV-file over time, how can this be approached? A small code example would be appreciated.

Thank you in advance

**Comment by pvinod13** [comment, marked as answer]
Hi,

Apologies for late reply. We are concentrating on development portal to answer these kinds of questions. 

As per my understanding, the easiest way to move the object is by changing the working co-ordinate system (https://aedt.docs.pyansys.com/release/0.6/examples/02-SBR%2B/SBR_Doppler_Example.html#sphx-glr-examples-02-sbr-sbr-doppler-example-py).  If the positions are available in the .csv file, update the pose values using the following command

        hfss.parametrics.add_from_file("csv_file_path/text file path")

**Please make sure the variable names are in-line with the CSV file headers.

If this is not working, please give more details to address the issue.

Thanks you in advance.
